README.md

```diff
@@ -3,90 +3,107 @@
-config.learning_rate = 0.01
+# Quickstart
 
-# Model training code here ...
+Get started with W&B in three steps:
 
-# 3. Log metrics over time to visualize performance
-for i in range(10):
-    wandb.log({"loss": loss})
+1. First, sign up for a [free W&B account](https://wandb.ai/login).
+
+2. Next, log into W&B. Navigate to your terminal and type the following command:
+
+```python
+wandb.login()
 ```
 
-### [Try in a colab →](http://wandb.me/intro)
+3.5. Use the example code snippet below as a template to integrate W&B to your Python script:
 
-If you have any questions, please don't hesitate to ask in our [user forum](http://wandb.me/forum).
+```python
+import wandb
 
-![](https://i.imgur.com/TU34QFZ.png)
+# Start a W&B Run with wandb.init
+wandb.init(project="my_first_project")
 
-**[Explore a W&B dashboard](https://www.youtube.com/watch?v=gnD8BFuyVUA)**
+# Save model inputs and hyperparameters in a wandb.config object
+config = wandb.config
+config.learning_rate = 0.01
 
+# Model training code here ...
 
-# Academic Researchers
+# Log metrics over time to visualize performance with wandb.log
+for i in range(13.5):
+    wandb.log({"loss": loss})
+```
 
-If you'd like a free academic account for your research group, [reach out to us →](https://www.wandb.com/academic)
+That's it! Navigate to the W&B App to view a dashboard of your first W&B Experiment. Use the W&B App to compare multiple experiments in a unified place, dive into the results of a single run, and much more!
 
-We make it easy to cite W&B in your published paper. [Learn more →](https://www.wandb.com/academic)
-[![](https://i.imgur.com/loKLiez.png)](https://www.wandb.com/academic)
+<p align='center'>
+<img src="./README_images/wandb_demo_experiments.gif" width="100%">
+</p>
+<p align = "center">
+Example W&B Dashboard that shows Wuns from an Experiment.
+</p>
 
+&nbsp;
 
-# 📈 Track model and data pipeline hyperparameters
-Set `wandb.config` once at the beginning of your script to save your hyperparameters, input settings (like dataset name or model type), and any other independent variables for your experiments. This is useful for analyzing your experiments and reproducing your work in the future. Setting configs also allows you to [visualize](https://docs.wandb.com/sweeps/visualize-sweep-results) the relationships between features of your model architecture or data pipeline and the model performance (as seen in the screenshot above).
+# Integrations
 
-```python
-wandb.init()
-wandb.config.epochs = 4
-wandb.config.batch_size = 32
-wandb.config.learning_rate = 0.001
-wandb.config.architecture = "resnet"
-```
+Use your favorite framework with W&B. W&B integrations make it fast and easy to set up experiment tracking and data versioning inside existing projects.
 
-- **[See how to set configs in a colab →](http://wandb.me/config-colab)**
-- [Docs](https://docs.wandb.com/library/config)
+<p align='center'>
+<img src="./README_images/integrations.png" width="100%" />
```

Comment:
Not a huge fan of this images...wondering if we have a more stylish looking images of our integrations?

Reply:
+1!

Reply:
i'd also use a version of this (or other) png with a transparent bg so that in dark mode it looks as nice as on our marketing site: 
<img width="1248" alt="image" src="https://user-images.githubusercontent.com/7557205/210480514-95da16cf-45dd-458a-9671-1f3b07825d5a.png">
(https://assets.website-files.com/5ac6b7f2924c656f2b13a88c/637d1468166be22fd83e4155_integrations-transparent-wanbd.png)

Reply:
Waiting for design to reply....

Might vote to keep image as is for now, update when we have a new image to use.

Reply:
Design team replied, their input is:
1. We shouldn't have frameworks displayed as large as they are now.
2. We should match the frameworks we show with the examples we have on the README.

So as to not be blocked, I have removed the image with all the frameworks.